src/Umbraco.Web.UI.Client/src/common/directives/components/tags/umbtagseditor.directive.js

```diff
@@ -189,7 +189,7 @@
          */
         function validateMandatory() {
             return {
-                isValid: !vm.validation.mandatory || (vm.viewModel != null && vm.viewModel.length > 0),
+                isValid: !vm.validation.mandatory || (vm.viewModel != null && vm.viewModel.length > 0) || (vm.value != null && vm.value.length > 0),
```

Comment:
This is not the issue. The issue is because the `validateMandatory `is not called on change in the input field,

If you change `ng-model="vm.viewModel.length"` to `ng-model="model.value"` in `umb-tags-editor.html`, then it looks like it works..

Reply:
Using you solution it still validates as empty the if if was empty on init.

Reply:
Yep, you are correct @bergmania, I must already had a tag saved and that's why it worked for me.
Added a line of code that allows re-validation on value changed, that should do the trick!